requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+coverage==4.5.1
+coveralls==1.3.0
+flake8==3.0.4
+pytest==3.5.1
+pytest-cov==2.5.1
+requests-toolbelt==0.8.0
```

Comment:
@gautamjajoo I am not sure whether we need this package or not. Can you please install CLI without using this package & check if it works fine.

Reply:
@RishabhJain2018 I tried doing that, but was not successful. I think so we need this package.

Reply:
@RishabhJain2018 Yeah. It works fine without it.